Neo4j.Driver/Neo4j.Driver/Internal/IO/MessageSerializers/FailureMessageV1Serializer.cs

```diff
@@ -21,10 +21,27 @@
 
 namespace Neo4j.Driver.Internal.IO.MessageSerializers
 {
-    internal class FailureMessageSerializer : ReadOnlySerializer
+    internal class FailureMessageV1Serializer : ReadOnlySerializer
```

Comment:
Would it be better to name this FailureMessageV3 serializer to match with the protocol naming conventions.

Reply:
Okay, will do.